SPECS/selinux-policy/selinux-policy.spec

```diff
@@ -275,13758 +290,3 @@ exit 0
   Fedora's policy and versioning tracks their policy fork specificially, whereas this tracks the upstream
   policy that Fedora's policy is based on.
 - License verified
-
-* Wed Oct 09 2019 Lukas Vrabec <lvrabec@redhat.com> - 3.14.4-37
```

Comment:
It might be GitHub truncating the diff, but it looks like you removed almost all of the changelog. Could you bring it back?

Reply:
Even though it has nothing to do with this policy? The Fedora policy has been a hard fork of this policy's upstream for years. The versions are also incomparable.

Reply:
Discussed offline, fine to remove the history, considering how heavily the original was modified and so we're closer to using Fedora's spec as guidance, than to a fork of it.